client/src/pages/Game.js

```diff
@@ -1,20 +1,17 @@
-import React, { useState } from "react";
+import React, {useState} from "react";
 // import { useHistory } from "react-router-dom";
 import questions from "../Json/questions.json";
 import "../css/game.css";
 import Column from "../components/Column/Column";
 import JeopardyModal from "../components/Modal/Modal";
 
 function Game() {
-  const [currQuestionID, setCurrQuestion] = useState(questions[0].id);
-  //game should be in charge of keeping track of score
-  const [show, setShow] = useState(false);
+  const [currQuestionID, setCurrQuestion] = useState(undefined);
 
```

Comment:
Another example of state that doesn't need to be tracked because it's derivable from other state.

You always set show to true when a question is clicked. `show` is always `false` when `currQuestionId` is `undefined`, and `show` is always `true` when `currQuestionId` is NOT `undefined`. Therefore tracking it in state is redundant. Whenever we need the valeu for `show` we can just do `show = currQuestionId != null`.